src/main/resources/i18n/Enums_en.properties

```diff
@@ -36,6 +36,27 @@ accelerator.SubmissionStatus.Rejected=Rejected
 accelerator.VoteOption.Conditional=Conditional
 accelerator.VoteOption.No=No
 accelerator.VoteOption.Yes=Yes
+docprod.DocumentStatus.Draft=Draft
+docprod.DocumentStatus.Locked=Locked
+docprod.DocumentStatus.Published=Published
+docprod.DocumentStatus.Ready=Ready
+docprod.DocumentStatus.Submitted=Submitted
+docprod.VariableInjectionDisplayStyle.Block=Block
+docprod.VariableInjectionDisplayStyle.Inline=Inline
+docprod.VariableTableStyle.Horizontal=Horizontal
+docprod.VariableTableStyle.Vertical=Vertical
+docprod.VariableTextType.MultiLine=Multi Line
+docprod.VariableTextType.SingleLine=Single Line
+docprod.VariableType.Date=Date
+docprod.VariableType.Image=Image
+docprod.VariableType.Link=Link
+docprod.VariableType.Number=Number
+docprod.VariableType.Section=Section
+docprod.VariableType.Select=Select
+docprod.VariableType.Table=Table
+docprod.VariableType.Text=Text
+docprod.VariableUsageType.Injection=Injection
+docprod.VariableUsageType.Reference=Reference
```

Comment:
I wonder if we should make these non-localizable since I don't see us exposing variable manifests directly to end users any time soon. No harm making them localizable, though.

Reply:
Good point. I think I'd rather just leave them here if that's alright since the work is already done.